### ISSUE TITLE
Double-clicking Fiji future launcher doesn't work on arm64 mac

### ISSUE THREAD

**Issue by hinerm** [fiji/fiji#370]
I ran the `config/fix-app.sh` script and can run from the command line, but the launcher silently crashes when I double-click (even with https://github.com/fiji/fiji/issues/369 fixed locally)

**Comment by ctrueden** [comment]
There is a problem with the macOS bundle's included OpenJDK; I did not preserve symlinks in the zip file. We need to make a new bundle with the JDK symlinks done correctly. For the moment, try downloading a fresh Zulu OpenJDK 21 and replacing the one in `Fiji-Future.app/java`.

**Comment by hinerm** [comment]
>For the moment, try downloading a fresh Zulu OpenJDK 21 and replacing the one in Fiji-Future.app/java

@ctrueden no luck. Double-clicking still doesn't work

**Comment by ctrueden** [comment]
Since this issue was filed, the .app structure of Fiji is being revised to facilitate successful code signing and notarization. Next step is to upload a nested `Fiji.app` to the Java-8 update site (removing the Jaunch-specific stuff from the bare `Contents` folder). Then @hinerm you can update your classic Fiji, drop in an arm64 OpenJDK into `macos-arm64` (or `macosx-arm64`; either should work at least on the Jaunch side), and then double click the nested Fiji.app. If it launches, we'll be able to close this!

**Comment by ctrueden** [comment]
Blocked by #235.

**Comment by ctrueden** [comment]
Closing in favor of #385.
